source/Databricks/source/SqlStatementExecution/DatabricksSqlStatementExecutionExtensions.cs

```diff
@@ -15,48 +15,16 @@
 using System;
 using System.Net.Http;
 using System.Net.Http.Headers;
-using Energinet.DataHub.Core.Databricks.SqlStatementExecution.Abstractions;
-using Energinet.DataHub.Core.Databricks.SqlStatementExecution.Client;
-using Energinet.DataHub.Core.Databricks.SqlStatementExecution.Configuration;
-using Energinet.DataHub.Core.Databricks.SqlStatementExecution.Constants;
-using Energinet.DataHub.Core.Databricks.SqlStatementExecution.ResponseParsers;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Options;
 
-namespace Energinet.DataHub.Core.Databricks.SqlStatementExecution.Extensions.DependencyInjection
+namespace Energinet.DataHub.Core.Databricks.SqlStatementExecution
```

Comment:
The `Energinet.DataHub.Core.Databricks.SqlStatementExecution.Extensions.DependencyInjection` namespace was used to resemble the structure in the [App package](https://github.com/Energinet-DataHub/geh-core/tree/main/source/App/source/FunctionApp/Extensions/DependencyInjection). Maybe we should keep it like that?
```suggestion
namespace Energinet.DataHub.Core.Databricks.SqlStatementExecution.Extensions.DependencyInjection
```

Reply:
Maybe - I am not sure. Does it add value to the user?

I would like to see namespaces that communicate 'behaviour' rather than technicality.

Reply:
I can see that other packages does not follow this pattern (e.g. Messaging) - with that in mind I will close this conversation and merge.